include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3329,6 +3331,38 @@ struct OrtApi {
   ORT_API2_STATUS(AddExternalInitializers, _In_ OrtSessionOptions* options,
                   _In_reads_(input_len) const char* const* initializer_names,
                   _In_reads_(input_len) const OrtValue* const* initializers, size_t initializers_num);
+
+  ORT_API2_STATUS(CreateAttribute,
+                  _In_ const char* name,
+                  _In_ const void* data,
+                  _In_ int len,
+                  _In_ ONNXTensorElementDataType type,
+                  _In_opt_ int is_array,
+                  _Outptr_ OrtOpAttr* op_attr);
```

Comment:
Should this take `OrtOpAttr**` which makes it consistent with other `CreateXXX()` APIs ?